tests/Feature/LangTest.php

```diff
@@ -35,12 +41,48 @@ public function testLang(): void
 			$locale = $lang_test->get_locale();
 
 			foreach ($keys as $key) {
-				static::assertArrayHasKey($key, $locale, 'Language ' . $lang_test->code() . ' is incomplete.');
+				try {
+					static::assertArrayHasKey($key, $locale, 'Language ' . $lang_test->code() . ' is incomplete.');
+				} catch (ExpectationFailedException $e) {
+					$msgSection->writeln('<comment>Error:</comment> ' . $e->getMessage());
+					$failed = true;
+				}
 			}
 		}
 
 		static::assertEquals('en', Lang::get_code());
 		static::assertEquals('OK', Lang::get('SUCCESS'));
+		$this->assertFalse($failed);
+	}
+
+	/**
+	 * Test Languages are strictly.
+	 *
+	 * @return void
+	 */
+	public function testLangOverflow(): void
+	{
+		$msgSection = (new ConsoleOutput())->section();
+
+		$failed = false;
+
+		$full = Lang::get_lang();
+		$lang_available = Lang::get_lang_available();
+
+		foreach ($lang_available as $code) {
+			$lang_test = Lang::factory()->make($code);
+			$keys = array_keys($lang_test->get_locale());
+
+			foreach ($keys as $key) {
+				try {
+					static::assertArrayHasKey($key, $full, 'Language ' . $lang_test->code() . ' as too many keys.');
```

Comment:
```suggestion
					static::assertArrayHasKey($key, $full, 'Language ' . $lang_test->code() . ' has too many keys.');
```

Would it make sense to also print the extra key? Or is that done already?